src/App.js

```diff
@@ -1,24 +1,14 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import Input from "./components_redux/Input"
+import Form from "./components_redux/Form"
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <Input/>
+      <br></br>
```

Comment:
`br` is for a line break. If you want a space between different elements, you should use `margin` (css). Design vs content.

Reply:
I'll fix this for the next version.